README.md

```diff
@@ -120,6 +121,9 @@ All templates must be YAML formatted. You can also use ERB. The following local
 You can add additional variables using the `--bindings=BINDINGS` option. For example, `kubernetes-deploy my-app cluster1 --bindings=color=blue,size=large` will expose `color` and `size` in your templates.
 
 
+### Customizing behaviour with annotations
+
+- `kubernetes-deploy.shopify.io/timeout-override` (any resource): Override the tool's hard timeout for one specific resource. Both full ISO8601 durations and the time portion of ISO8601 durations are valid. Value must be between 1 second and 24 hours. Example values: 45s / 3m / 1h / PT0.25H
```

Comment:
How about adding: "This feature does not work as expected for the `deployment` resources because progressDeadlineSeconds takes precedence over hard timeouts and have a server side default value for deployments."

Reply:
Updated with more details on compatibility